### PR TITLE
dispute info improvements

### PR DESCRIPTION
**Pull request by epheph**

### CODE REVIEW

**Comment by epheph** [comment]
@bthaile adds disputeRound, and a lot more values inside stakeDetails (totalStake, completedStake, currentStake). Also calculates `size` for payouts that are not tentative, but also don't have a current crowdsourcer.

**Comment by epheph** [comment]
```
$ node test-client.js payloads/getDisputeInfo.json 

payloads/getDisputeInfo.json
{
  "id": 61,
  "jsonrpc": "2.0",
  "method": "getDisputeInfo",
  "params": {
    "marketIDs": ["0x0000000000000000000000000000000000000011","0x0000000000000000000000000000000000000211"]
  }
}

{
  "id": 61,
  "result": [
    {
      "marketID": "0x0000000000000000000000000000000000000011",
      "stakes": [
        {
          "isInvalid": false,
          "payout": [
            0,
            2
          ],
          "size": "39898",
          "currentStake": "0",
          "totalStake": "102",
          "completedStake": "102",
          "tentativeWinning": false
        },
        {
          "isInvalid": true,
          "payout": [
            1,
            1
          ],
          "totalStake": "20000",
          "completedStake": "20000",
          "tentativeWinning": true
        }
      ],
      "disputeRound": 1
    },
    {
      "marketID": "0x0000000000000000000000000000000000000211",
      "stakes": [
        {
          "isInvalid": false,
          "payout": [
            10000,
            0
          ],
          "size": "204",
          "currentStake": "0",
          "totalStake": "0",
          "completedStake": "0",
          "tentativeWinning": false
        },
        {
          "isInvalid": false,
          "payout": [
            0,
            10000
          ],
          "totalStake": "102",
          "completedStake": "102",
          "tentativeWinning": true
        },
        {
          "isInvalid": true,
          "payout": [
            5000,
            5000
          ],
          "size": "204",
          "currentStake": "0",
          "totalStake": "0",
          "completedStake": "0",
          "tentativeWinning": false
        }
      ],
      "disputeRound": 0
    }
  ],
  "jsonrpc": "2.0"
}

```

File: src/server/getters/get-dispute-info.ts [hunk 1]
```diff
@@ -1,77 +1,117 @@
...
+    const completedStakes = completedStakeByPayout[payout.payoutID];
+    const activeCrowdsourcer = activeCrowdsourcerByPayout[payout.payoutID];
+
+    const completedStakeAmount = new BigNumber(completedStakes == null ? 0 : completedStakes.amountStaked);
+    const totalStakeOnPayout = completedStakeAmount.add(new BigNumber(activeCrowdsourcer == null ? 0 : activeCrowdsourcer.amountStaked));
+
+    let currentAmounts: { size?: string; currentStake?: string };
+    if (payout.tentativeWinning === 1 || !isActiveMarketState(marketRow.reportingState)) {
+      currentAmounts = {};
+    } else if (activeCrowdsourcer == null) {
```
Comment (bthaile) [inline src/server/getters/get-dispute-info.ts]: if there is a tentativeWinning I would think there is a current amount staked. tentative winning should be set after initial reporter. Is it just the designated reporter stake? if only DR reported.

Comment (epheph) [reply src/server/getters/get-dispute-info.ts]: This branch here only provides the size (for current dispute) and currentStake (on that dispute). totalStakeOnPayout is calculated above, and for every payout

**Comment by epheph** [comment]
@bthaile did my response answer your question?

**Comment by bthaile** [comment]
Yes, answers my question.

**Pull request merged**